tools/docker/daml-all-in-one/Readme.md

```diff
@@ -0,0 +1,30 @@
+# DAML All in One Image
+
+An all in one DAML docker image with the `sample ledger contracts`.
+- This docker image is for `testing` and `development` only.
+- **Do NOT use in production!**
+
+## Build an image locally
+
+To build the besu v1.5.1 image locally, use:
```

Comment:
@raynatopedrajeta Please do a once-over of the documentation here to avoid copy-paste mistakes like talking about Besu here instead of DAML.